F logger tes

evgeniimv opened the pull request:
added logger to tes-module. Updated build.gradle and also added new xml file as logger config

evgeniimv committed:
added new module "tes-adapter" in root directory "cloud-pipeline"

evgeniimv committed:
finished sub-task 2759. Added basic Spring Web-Application in tes-adapter module. Create standalone JAR with "OK" response in endpoint "/v1/tasks/service-info" of URL

evgeniimv committed:
impl logger to tes-module. Updated build.gradle and added logger-xml config to resources.

mzueva commented on tes-adapter/src/main/resources/log4j2-spring.xml:
```diff
@@ -0,0 +1,86 @@
+        <Logger name="com.epam.pipeline" level="debug" additivity="true">
+            <AppenderRef ref="App-Appender"/>
+        </Logger>
+        <Logger name="org.springframework.security.saml" level="debug" additivity="false">
```
Remove this Logger, since we do not have `saml` module in this project

evgeniimv replied:
fixed.
I also changed Logger-name com.epam.pipeline to com.epam.pipeline.tesadapter in previous Logger.
Is it correct decision?

mzueva commented on tes-adapter/src/main/resources/log4j2-spring.xml:
```diff
@@ -0,0 +1,86 @@
+                </pattern>>
+            </PatternLayout>
+        </Console>
+        <RollingFile  name="App-Appender" fileName="${log-path}/pipeline.log"
```
Rename `pipeline.log` to `tes-adapter.log`

evgeniimv replied:
fixed

evgeniimv committed:
removed Logger for saml, via unnecessary reasons. Changed Logger name com.epam.pipeline to com.epam.pipeline.tesadapter

evgeniimv committed:
Merge branch 'tes-support' into f_logger_tes

evgeniimv committed:
Delete TesAdaptorsController

I pushed twice unnecessary file, so I removed it, because we already have proper Controller
